backend/database/seed.go

```diff
@@ -47,9 +47,9 @@ func SeedTierlists() error {
 	}
 
 	tierlists := []models.Tierlist{
-		{UserID: 1},
-		{UserID: 2},
-		{UserID: 3},
+		{Name: "Tierlist1", Description: "This is tierlist 1", CreatorID: 1},
```

Comment:
Check out this section of the docs, https://gorm.io/docs/create.html#Create-With-Associations

Will allow you to create a single Tierlist object using the Items / Tiers properties. Gorm can then look at this object and map that to SQL for you. Rather than having to call different functions to seed the Tierlist, Items, Tiers and then link together by a hardcoded ID if you create them from a single object Gorm will ensure they're associated to each other when they're inserted.

Is a much more intuitive way to work with an ORM and worth getting used to.